### ISSUE TITLE
hitlist grey list

### ISSUE THREAD

**Issue by dolmen** [CPAN-PRC/cpan-prc.github.io#4]
Some distributions are in the [hitlist](http://cpan-prc.org/hitlist.html) because they don't have a Github repo. But some of them have a public repo, but it is just not on GitHub. For example, [Net-SSLeay](http://anonscm.debian.org/viewvc/net-ssleay/).
Should we really blame the author for not endorsing GitHub? I think it would be better to concentrate the efforts first on distributions that have no public repo at all or just not advertised. The distributions that have a repo advertised, but not on Github would appear on a separate, "grey", list.

**Comment by neilb** [comment]
Yeah, should probably at least highlight the dists that do have a repository set, when it's not a github repo.

This is the _PRC_ hit list, so I think they should stay on there. For example someone commented on the fact that most of ADAMK's dists have a repo specified, but it's not one that encourages people to contribute.

I'll do a summary table of where people have repos, since that'd be interesting too.
